packages/node_modules/@ciscospark/react-component-activity-share-files/src/index.js

```diff
@@ -65,19 +73,23 @@ export function ActivityShareFiles(props) {
       }
     }
 
-    return (
-      <FileShareDisplay
-        actor={actor}
-        file={file}
-        isFetching={isFetching}
-        isPending={isPending}
-        key={file.url}
-        objectUrl={objectUrl}
-        onDownloadClick={onDownloadClick}
-        type="chat"
-        timestamp={timestamp}
-      />
-    );
+    if (file.type !== 'cardReference') {
```

Comment:
Looks like you meant to use the constant here.

Reply:
made it as constant